draft-ietf-masque-connect-ip.md

```diff
@@ -1392,9 +1392,11 @@ based on these markings. However, the use of HTTP can limit the possibilities
 for differentiated treatment of the tunneled IP packets on the path between the
 IP proxying endpoints.
 
-If tunneled packets are subject to congestion control by the outer connection,
-the tunneled packets need to be treated equally regardless of their DSCP
-markings to not disrupt the congestion controller. In this scenario, the IP
+When a connection is congestion-controlled, marking packets with different DSCP
+can lead to reordering between them, and that can in turn lead the congestion
+controller to perform poorly. If tunneled packets are subject to congestion
+control by the outer connection, they need to avoid carrying different DSCP
```

Comment:
Do we want to clarify that the possible solutions to this are multiple HTTP/3 connections with independent CONNECT-IP sessions?

Reply:
Isn't that already there further down in that paragraph?

Reply:
Yes, you're right, but I had to read it 3 times to notice. I think it gets missed with the note to scope on particular DSCP values. Maybe let's add a newline and separate that out into its own paragraph, since it's kind of a separate thought?